mariadb-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -93,6 +94,27 @@ prepare() {
     chmod 600 $config $command
 }
 
+# Read MYSQLD_CONFIG envvar and write all settings separated by ';' character.
+# E.g. MYSQLD_CONFIG="skip-networking;max_allowed_packet=16M" will produce:
+#
+# ```
+# [mysqld]
+# skip-networking
+# max_allowed_packet=16M
+# ```
+prepare_mysqld_config() {
+    if [ -z $MYSQLD_CONFIG ]; then
+        return
+    fi
+    local mysqld_env_config_file="$mysqld_config_splitted/environment-$(uuidgen).cnf"
+    echo "# This configuration has been dynamically generated based on MYSQLD_CONFIG envvar contents" > $mysqld_env_config_file
```

Comment:
I would `echo` the config options we are passing for debugging purposes. Otheriwise, setting `MYSQLD_CONFIG=garbage` would result in the container not starting and a `docker logs` couldn't help either...

Reply:
I guess `docker logs` will print some `mysqld` error if that `garbage` prevents it from start. In any case, I'm adding the output too.

Reply:
Done.